src/Command/RefreshCommand.php

```diff
@@ -51,54 +99,54 @@ public function configure()
         $this->addOption('session', null, InputOption::VALUE_OPTIONAL, 'The session to use for this command');
     }
 
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        parent::interact($input, $output);
+
+        // TODO: check if it is phpcr check for session or if it is orm do not accept session
+    }
```

Comment:
Is this necessary? Can we remove it? If this a foundation for another feature, then it can be another PR.